bin/juxtaposer/deploy/secretless.yml

```diff
@@ -2,63 +2,69 @@ version: 2
 
 services:
   mysql-tcp:
-    connector: mysql
+    protocol: mysql
```

Comment:
can we switch back to `connector` please? we're moving to this standard everywhere.